src/server/game/Entities/Player/Player.h

```diff
@@ -1660,13 +1660,9 @@ class Player : public Unit, public GridObject<Player>
         void SetTarget(ObjectGuid const& /*guid*/) override { } /// Used for serverside target changes, does not apply to players
         void SetSelection(ObjectGuid const& guid) { SetGuidValue(UNIT_FIELD_TARGET, guid); }
 
-        uint8 GetComboPoints() const { return m_comboPoints; }
```

Comment:
@sarnex: I'm not quite sure what you are referring too but this is not a contest.